docs/user-guide/cli-using-creating-profiles.md

```diff
@@ -0,0 +1,246 @@
+# Creating profiles
+
+Configuration profiles are used to connect to different mainframe services, and you can structure each profile based on how that connection is made.
+
+For example, profiles can be nested to share the same connection information, or keep different information separate. Services can be accessed directly by Zowe CLI, or they can be accessed through the API Mediation Layer.
+
+Review the following scenarios to help determine the profile types best suited for your environment.
+
+As a team leader, you can share the configuration you create with your team members so they can easily access mainframe services.
+
+## Accessing LPARs that contain services that share the same credentials
+
+In the following configuration, nested profiles use the credentials from the same base profile to access services directly on multiple LPARs: **[is this explanation correct?]**
+
+```
+{
+    "$schema": "./zowe.schema.json",
+    "profiles": {
+        "lpar1": {
+            "properties": {
+                "host": "example1.com"
+            },
+            "profiles": {
+                "zosmf": {
+                    "type": "zosmf",
+                    "properties": {
+                        "port": 443
+                    }
+                },
+                "tso": {
+                    "type": "tso",
+                    "properties": {
+                        "account": "ACCT#",
+                        "codePage": "1047",
+                        "logonProcedure": "IZUFPROC"
+                    }
+                },
+                "ssh": {
+                    "type": "ssh",
+                    "properties": {
+                        "port": 22
+                    }
+                }
+            }
+        },
+        "lpar2": {
+            "properties": {
+                "host": "example2.com"
+            },
+            "profiles": {
+                "zosmf": {
+                    "type": "zosmf",
+                    "properties": {
+                        "port": 1443
+                    }
+                }
+            }
+        },
+        "base": {
+            "type": "base",
+            "properties": {
+                "rejectUnauthorized": true
+            },
+            "secure": [
+                "user",
+                "password"
+            ]
+        }
+    },
+    "defaults": {
+        "zosmf": "lpar2.zosmf",
+        "tso": "lpar1.tso",
+        "ssh": "lpar1.ssh",
+        "base": "base"
+    },
+    "autoStore": true
+}
+```
+## Accessing LPARs that contain services that do not share the same credentials
+
+In the following configuration, nested profiles use the credentials from different service profiles to access services directly on multiple LPARs.
```

Comment:
I might suggest changing this to something like:
"In the following configuration, nested profiles use credentials defined **at the LPAR level** to access services directly on multiple LPARs."

I'm not sure how that should be worded, but I think it is worth emphasizing that this config only defines one set of credentials for each LPAR (not using credentials from the service profiles).

Reply:
@adam-wolfe: The reason I mentioned service profiles is b/c I wanted to emphasize the difference between the first example, which uses a base profile for creds, and the second. My understanding is that b/c the LPARs use different creds, we now have to pull the creds from a single shared base profile and store the cred information in separate service profiles, which, yes, are defined at the LPAR level. 
Would it be correct if I rewrite it to say: " ... nested profiles use the credentials from **service profiles in different LPARs** to access services directly on multiple LPARs."
Or am I off base?

Reply:
I guess I'm not sure about how we use the term "service profile". The LPARs in this config are defined as profiles, but they don't define connections to a particular service. They're like a profile that contains multiple service profiles (like zosmf, tso, ssh).

Reply:
Resolving this convo as we've discussed this issue w/ the squad and have agreed to start using the new term "parent profile" to refer to cases like these.